NEWS.md

```diff
@@ -1,3 +1,66 @@
+# New in snapd 2.64:
+* Support building snapd using base Core22 (Snapcraft 8.x)
+* FIPS: support building FIPS complaint snapd variant that switches to FIPS mode when the system boots with FIPS enabled
+* AppArmor: update to AppArmor 4.0.1
+* AppArmor: support AppArmor snippet priorities
+* AppArmor prompting: add path pattern matching
+* AppArmor prompting: support constraints and abstract permissions
```

Comment:
The most important (user-facing) changes are these:
* AppArmor prompting: add checks for prompting support, include support/flag enablement in system key, and restart snapd if prompting flag changes
* AppArmor prompting: include prompt prefix in AppArmor rules if prompting is supported and enabled

The full set of AppArmor prompting-related changes could also include:
* AppArmor prompting: add common types, constraints, and mappings from AppArmor permissions to abstract permissions
* AppArmor prompting: add path pattern parsing and matching

But the first two are the most important, and should replace what's currently listed if you're tight on space.

Reply:
thanks

Reply:
fixed